src/lints/enum_repr_int_removed.ron

```diff
@@ -18,7 +18,12 @@ SemverQuery(
                         name @tag @output
 
                         attribute {
-                            value @filter(op: "regex", value: ["$repr_regex"])
```

Comment:
Can we output the `as_string` value here? It can really help with debugging, even though it isn't in the templated error message, because the new `-vv` "extra verbose" CLI option outputs all the values returned by a semver lint.